conda/gateways/logging.py

```diff
@@ -90,6 +90,13 @@ def trace(self, message, *args, **kwargs):
     if self.isEnabledFor(TRACE):
         self._log(TRACE, message, args, **kwargs)
 
-
 logging.addLevelName(TRACE, "TRACE")
 logging.Logger.trace = trace
+
+
+def warn(self, message, *args, **kwargs):
+    if self.isEnabledFor(WARN):
+        self._log(WARN, message, args, **kwargs)
+
+logging.addLevelName(WARN, "WARN")
+logging.Logger.warn = warn
```

Comment:
No need for the custom method and level name,
```python
# suppress DeprecationWarning for warn method
# FIXME: remove this when all calls -- possibly in _vendor libs -- to warn are replaced
logging.Logger.warn = logging.Logger.warning
```
should suffice. (_vendor libs includes `auxlib` ;) )